package com.exceptions;

public class Lab564 {

	public static void main(String[] args) {
		System.out.println("Main started");
//		Exception in thread "main" java.lang.ArithmeticException: / by zero
		int x=10/0;
		System.out.println("Main end");	}

}
